### ISSUE TITLE
Mass Fabricator not working [1.19.2]

### ISSUE THREAD

**Issue by MarwenX3M** [TinyModularThings/IC2Classic#1144]
oops just hit enter gonna add some text now

**Comment by MarwenX3M** [comment]
So I am trying to use mass fabricator and its not working. I am giving it power but it does not use it. The red lightning bolt fills with power but it does not do anything, and I have added som scrap for amplifier too. Tried giving it redstone power too. Please help if I am doing something wrong or maybe this is a bug.

**Comment by CrossVas** [comment]
Could you show the actual setup you're using?

**Comment by MarwenX3M** [comment]
Okay Im gonna add screenshots

**Comment by MarwenX3M** [comment]
![2024-04-12_15 37 09](https://github.com/TinyModularThings/IC2Classic/assets/101430664/31237775-0b24-40d2-9dbc-f94c5f60592f)
![2024-04-12_15 37 22](https://github.com/TinyModularThings/IC2Classic/assets/101430664/2383e8b5-ffc2-4820-bcb9-34ca2a66078f)

So in the first picture is a ESU with full battery connected to a HV-transformer and then that is connected to a mass fabricator. Also behind is a recycler that is making scrap which is sent to a drawer and that is sent with a modular router to the mass fab.

**Comment by Speiger** [comment]
@MarwenX3M does the transformer face the ESU?
Also is the transformer not redstone powered?

**Comment by MarwenX3M** [comment]
Yes they are facing correct, I wrenched to check and make sure. And yes no redstone signal to the transformer. When I replace the mass fab it charges up so it seems to get the power, just not using it

**Comment by CrossVas** [comment]
An EU Reader in your hotbar will help understand what's going on with the energy, because the `red lightning bolt` is there by default, it's not an actual indicator of energy level.

**Comment by MarwenX3M** [comment]
Yeah the EU-reader says 0 so no throughput of energy is happening

**Comment by CrossVas** [comment]
Check the transformer with the EUReader in your hotbar, is it `Inverted`?

**Comment by MarwenX3M** [comment]
Okay so I think I figured it out now. It is the ESU. Despite it visually facing the right direction it was de facto outputting energy on the other side. This makes sense because sometimes when I load the world it would face the other way than I left it as, making me have to adjust it continously. So seemingly the issue is that the de facto output side of the ESU did not match the side you wrench it too. I should mention that I used a crescent hammer from the thermal mod to turn the ESU which might be the cause of the issue here. In the picture here I supply, the mass fabricator is working strangely enough. 

![2024-04-12_16 18 18](https://github.com/TinyModularThings/IC2Classic/assets/101430664/4c75733a-f3bf-4a79-9d99-e47a94b7d94e)
